cmd/prometheus/main_test.go

```diff
@@ -14,11 +14,98 @@
 package main
 
 import (
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"testing"
+	"time"
 
 	"github.com/prometheus/prometheus/util/testutil"
 )
 
+var promPath string
+var promConfig = filepath.Join("..", "..", "documentation", "examples", "prometheus.yml")
+
+func TestMain(m *testing.M) {
+	var err error
+	promPath, err = os.Getwd()
+	if err != nil {
+		fmt.Printf("can't get current dir :%s \n", err)
+		os.Exit(1)
+	}
+	promPath = filepath.Join(promPath, "prometheus")
+
+	flag.Parse()
+	if !testing.Short() {
```

Comment:
Can we have 
```
if testing.Short() {
  t.Skip()
}
```

Reply:
nice catch about the /data folder , I forgot about this , thanks.